script/common.bash

```diff
@@ -41,11 +41,21 @@ get_go_version() {
 setup_go_root() {
   local version=${1}
 
-  # Setup GOROOT and add go to the PATH.
-  GIMME=${_sdir}/gimme/gimme
-  debug "Gimme version $(${GIMME} version)"
-  ${GIMME} "${version}" > /dev/null
-  source "${HOME}/.gimme/envs/go${version}.env" 2> /dev/null
+  # Use the current Go installation if the given Go version is already
+  # installed and configured.
+  if command -v go &>/dev/null ; then
```

Comment:
IMPORTANT: stdout/stderr redirection otherwise the output can affect other commands such as: 

```
# Ensure the Go version in .go_version is installed and used.
GOROOT?=$(shell ./script/run_with_go_ver go env GOROOT)
```
 in the toplevel makefile